reproducer.yml

```diff
@@ -57,11 +56,18 @@
       ansible.builtin.import_role:
         name: repo_setup
         tasks_from: rhos_release.yml
-
   roles:
     - role: ci_setup
-    - role: reproducer
 
+- name: Prepare switches if present
```

Comment:
```yaml
vars:
  has_switches: "{{ groups['switches'] is defined }}"
```

then you can check on that `has_switches` as a condition to early end_play in the playbook itself. Doing so would allow to still run the play as a hook by passing the boolean.